tests/scripts/thread-cert/Cert_7_1_04_BorderRouterAsRouter.py

```diff
@@ -108,6 +115,44 @@ def test(self):
             if addr[0:10] == '2001:2:0:1' or addr[0:10] == '2001:2:0:2':
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        SED = pv.vars['SED']
+        _rpkts = pkts.filter_wpan_src64(ROUTER)
+
+        # Step 3: The DUT MUST send properly formatted MLE Advertisements
+        # The DUT MUST send a CoAP Server Data Notification message
+        # with the server’s information (Prefix, Border Router) to the Leader
+        _rpkts.filter_mle_cmd(MLE_CHILD_ID_REQUEST).must_next()
+        _rpkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: {SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, ROUTE64_TLV} == set(p.mle.tlv.type))
+        _rpkts.filter_coap_request(SVR_DATA_URI).must_next()
+        _rpkts_med = _rpkts.copy()
+        _rpkts_sed = _rpkts.copy()
+
+        # Step 4: Automatically transmits a 2.04 Changed CoAP response to the DUT
+        # Step 5: The DUT MUST send a multicast MLE Data Response
+        _rpkts.filter_mle_cmd(MLE_DATA_RESPONSE).must_next().must_verify(
```

Comment:
Which step requires verification of a `MLE_DATA_RESPONSE` message?

Reply:
step 5